content-scripts/modules/utilities/popover.js

```diff
@@ -1,9 +1,5 @@
 let popoverId = 0;
 
-/**
- * @param {HTMLElement} popover
- * @param {HTMLElement} [target]
- */
 export const createPopover = (popover, target = popover) => {
```

Comment:
I believe this function is now deprecated.

Reply:
deleted!